Automattic/jetpack-onboarding
# Install WooCommerce during on-boarding

This PR adds checkboxes to the business address page to determine if folks want to sell online and install WooCommerce. The UI is rather basic right now, and you can see it in this screencast:

https://cloudup.com/c7dFAwYLiIt

To test it yourself:
- checkout this branch
- disconnect from jetpack and reset JPO options
- go through the JPO wizard, ensuring you select 'Business', 'I want to sell online', 'Install Woo now'
- ensure that WooCommerce is installed and you can access the Woo setup wizard when complete

cc: @jessefriedman @tyxla
